### ISSUE TITLE
Consider validating against PR titles instead of numbers

### ISSUE THREAD

**Issue by hectorsector** [githubtraining/write-github-script#7]
When validating a learner's action, leaving everything up to the pull request number puts the learner in a state that they are unable to recover from. For example, after the learner merges say PR 2, and let's assume Learning Lab misses that event (it did in my testing), the learner is then unable to move forward because they are told to merge PR 2. Instead, we can capture the title, and tell the learner that all the need to do is merge a PR with that title. 

https://github.com/githubtraining/write-github-script/blob/3396a674484501dd49d522d333af182930a6c59a/config.yml#L123-L134

**Comment by mattdavis0351** [comment]
@hectorsector you ran into this issue even with the store implemented?

**Comment by hectorsector** [comment]
Yes! This is an issue with too strict of a validation on the PR number. The example I described above happened to me. 

I guess our options are:
- do not validate against the PR number on a merge event
- maybe look for the right files in master instead? 
- validate on the title instead (using whatever title the learner chose)
